packages/nimble-components/src/table/tests/table.spec.ts

```diff
@@ -416,6 +416,7 @@ describe('Table', () => {
             element.setData(simpleTableData);
```

Comment:
@atmgrifter00, I'm moving your testing question here so that we can follow the flow of the discussion.

> Does it make sense to add tests where you update various parts of the table and assert that multiple, appropriate get<> methods are called?

I'm not against the idea of adding more tests, but we can't spy on the get<> methods since they are private. I could add a test (or tests) that verify changing multiple things actually updates the table correctly. I'm not sure that I'd be able to test that things are actually batched, but I think it would be pretty easy to verify that one change isn't overwriting the need to make a different change. Does that seem reasonable?